src/components/Pagination.js

```diff
@@ -30,56 +30,60 @@ const Pagination = (props) => {
     }
   }, [storyCount])
 
-  return (
-    <div className='pagination'>
-      <span
-        className={`btn btn-pagination ${
-          currNumber <= 1 ? 'btn-pagination-disabled' : ''
-        }`}
-        onClick={() => {
-          if (pages.find((page) => page === currNumber - 1)) {
-            setCurrNumber((currNumber) => currNumber - 1)
-            getPage(currNumber - 1)
-          }
-        }}
-      >
-        <EOS_KEYBOARD_ARROW_LEFT className='eos-icons eos-18' />
-        {`Prev`}
-      </span>
-      <div className='btn btn-pagination'>
-        {pages
-          ? pages.map((ele, key) => {
-              return (
-                <span
-                  className={`number ${currNumber === ele ? 'selected' : ''}`}
-                  onClick={() => {
-                    setCurrNumber(ele)
-                    getPage(ele)
-                  }}
-                  key={key}
-                >
-                  {ele}
-                </span>
-              )
-            })
-          : ''}
+  if (storyCount > 5) {
```

Comment:
can we change it to something like this {storyCount>5 && ( <jsx> ) } instead of if else? returning null doesnt make any sense